### PR TITLE
fix 404 page: use createOptimizedPicture for images

### LINKED ISSUE #138: [Page] 404 page should have correct image formats
404 pages currently only have webp images. We should habe jpeg as fallback as well and proper `<picture>` tag.

### PR DESCRIPTION
**Pull request by mtilburgadobe**
use createOptimizedPicture for images to add fallbacks for jpeg and create picture tag

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #138 

Test URLs:
- Before: https://main--eder-group--hlxsites.hlx.page/404.html
- After: https://issue-27--eder-group--hlxsites.hlx.page/404.html

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -91,8 +87,13 @@ <h6>SUCHE</h6>
     <div class="section sidebar contact-container press-content">
         <div>
             <h2> Kontakt </h2>
-            <div>
-              <img src="./images/contact.webp">
+            <div class="kontakt-pic">
+              <script type="module">
+                import { createOptimizedPicture } from './scripts/lib-franklin.js';
```
Comment (mhaack) [inline 404.html:92]: Import is done twice. Lets move the import to an `<script>` tag in `<head>`

**Review by mhaack — CHANGES_REQUESTED** [review]

File: 404.html (hunk 1 above)
Comment (mtilburgadobe) [reply 404.html:92]: Hi @mhaack - been trying several different possibilities here, problem is when I import it in <head> it's not available in <body>.  The import has to be in a script of type module, and some googling shows that references are not available cross modules in an html file...

Any ideas of how to achieve this?

Comment (alexcarol) [reply 404.html:92]: Why not include a js file via script tag instead?

Comment (mtilburgadobe) [reply 404.html:92]: @alexcarol - I tried this as well.  The function is defined in lib-franklin.js, but including this file also does not work(and probably not any better than importing just the function)

Comment (mtilburgadobe) [reply 404.html:92]: I have an idea, I will move it all into the same script at the end of <body>

Comment (mtilburgadobe) [reply 404.html:92]: ah yes, that worked

Comment (mtilburgadobe) [reply 404.html]: @mhaack / @alexcarol - please take another look

**Comment by mtilburgadobe** [comment]
found another issue, seems this breaks if there is a 404 page not at root, for example:

https://issue-27--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/doesnotexist

**Review by mhaack — APPROVED** [review]

**Pull request merged**